frontend/src/TooljetDatabase/Forms/RowForm.jsx

```diff
@@ -15,12 +15,16 @@ const RowForm = ({ onCreate, onClose }) => {
   const darkMode = localStorage.getItem('darkMode') === 'true';
   const { organizationId, selectedTable, columns } = useContext(TooljetDatabaseContext);
   const [fetching, setFetching] = useState(false);
-  const [activeTab, setActiveTab] = useState(Array.isArray(columns) ? columns.map(() => 'Custom') : []);
+  const [activeTab, setActiveTab] = useState(Array.isArray(columns) ? columns.map(() => 'Default') : []);
+
   const [inputValues, setInputValues] = useState(
     Array.isArray(columns)
-      ? columns.map((item) => {
-          if (item.column_default !== null) {
+      ? columns.map((item, index) => {
+          if (item.accessor === 'id') {
             return { value: '', checkboxValue: false, disabled: false };
+          }
+          if (item.column_default !== null) {
+            return { value: item.column_default || '', checkboxValue: item.column_default, disabled: true };
```

Comment:
Will fail when a user creates a row with bool column which has false as default value.

Based on boolean datatype you would have to set the checkbox value, string will always evaluate as true for bool otherwise